src/sentry/api/serializers/models/actor.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+
+import six
+from sentry.api.serializers import Serializer
+from sentry.models import User, Team
+
+
+class ActorSerializer(Serializer):
```

Comment:
As we discussed last week IRL, I think we should refactor this slightly to serialize `Actor` objects directly and not do this dance through User/Team here. It would also eliminate the need to have the `Actor.resolve_dict` that was added.

I won't block on this now, but I think we should follow up on it.